polonius-engine/src/output/datafrog_opt.rs

```diff
@@ -316,6 +362,16 @@ pub(super) fn compute<Region: Atom, Loan: Atom, Point: Atom>(
                 ((b, p), ())
             });
 
+            // borrow_live_at(B, P) :-
+            //   dead_borrow_region_can_reach_dead(R1, B, P),
+            //   subset(R1, R2, P),
```

Comment:
I believe this is a copy-paste typo since we're not using `subset` here (subsets being handled in `dead_borrow_region_can_reach_dead` already) ? 
So the following line may be `region_live_at(R1, P)`, and the closure argument `_r1` — but that would be almost too nitpicky (_but_ still not worthy of a "pacify the mercilous $reviewer" commit, I hope)

Reply:
Well, it depends. I am using the **intermediate variable** `dead_borrow_region_can_reach_dead_1`, which is equal to `dead_borrow_region_can_reach_dead(R1, B, P), subset(R1, R2, P)`. So yes you don't see `subset` below but the effect is still true to the datalog as written. Maybe I should comment *that*?

Reply:
Yeah and in any case we always document the datalog semantics and not the literal datafrog syntax as I was wrongly mentioning. The _1 was indeed subtle but regardless: you're absolutely right and my comment was misguided. 

The new comment is perfectly clear.